statchatexpress/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+package.json
```

Comment:
Thanks for adding this .gitignore. 

We actually want to commit both package.json + package-lock.json. This will let other people who clone your project run it on their machines.